package.json

```diff
@@ -18,8 +18,7 @@
     "test:playwright:mobile": "echo \"⭕️ Running playwright mobile tests...\" && cd playwright && PLAYWRIGHT_BASE_URL=https://www-stage.wellcomecollection.org NODE_ENV=test platform=mobile yarn test && echo \"✅️ playwright mobile tests finished\"",
     "test:all:unit": "yarn test:common && yarn test:content && yarn test:identity",
     "e2e-stage": "yarn test:playwright && yarn test:playwright:mobile",
-    "run-concurrently": "./scripts/run-concurrently.sh",
-    "run-concurrently:clean": "./scripts/run-concurrently.sh -c",
+    "config-local-apis": "./scripts/configure-local-running.sh",
```

Comment:
I think this name may have changed and we're trying to run `configure-local-apis.sh`?